manifests/source.pp

```diff
@@ -18,7 +18,7 @@
   $key_content       = undef,
   $key_source        = undef,
   $trusted_source    = undef,
-  $notify_update     = undef,
+  $notify_update     = true,
```

Comment:
This is due to https://tickets.puppetlabs.com/browse/PUP-6422 and does not actually need to be changed.